.gitignore

```diff
@@ -24,6 +24,7 @@ hs_err_pid*
 
 # Ballerina
 Ballerina.lock
+ballerina/Dependencies.toml
```

Comment:
We have to remove this from the gitignore and add this file to auto commit in `build.gradle` file

Reply:
corrected in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/249/commits/4623e4105225e6df24f3b99a9fc72ca0a63e7de1